schemas/v1.0/azure.yaml.json

```diff
@@ -79,6 +79,11 @@
                         "type": "string",
                         "title": "Path to the service source code directory"
                     },
+                    "dotnetProjectFile": {
```

Comment:
check below the key `allOf` and see if we can have a condition for this field to only be OK when the language is `dotnet`.

might be something like
```
"if": {
    "not":{
        "properties": {
            "language": {
                "const": "dotnet"
            }
       }
    }
},
"then": {
      "properties": {
          "dotnetProjectFile": false
      }
  }
```